src/exercises/9/exercise.js

```diff
@@ -2,6 +2,15 @@
 // - Use Array methods.
 // - Do not use loops
 
+//Here we will be using map method which it returns an new array with modified elemts in the result array
+//map method is simialr to forEach where forEach will not return any array as a result
 function multiplyArrayByTwo(inputArray) {
+  // inputArray = inputArray.map(function (value) {
+  //   return value * 2;
+  // });
+
+  //using array function in the map methods
+  inputArray = inputArray.map((value) => value * 2);
   console.log(`Given input array is ${inputArray}`);
```

Comment:
Print before and after arrays. Also do not mutate the array. Take the output into a new array.

Do you think map function mutates the array or creates a new one?

Reply:
map method will create a new array and return it